Cargo.toml

```diff
@@ -14,7 +14,7 @@ gethostname = "0.5.0"
 log = "0.4.22"
 prost = "0.13.3"
 prost-types = "0.13.3"
-pyo3 = {version = "0.22.3", features = ["extension-module"]}
+pyo3 = {version = "0.24", features = ["extension-module"]}
```

Comment:
This is indeed otherwise trying examples in https://pyo3.rs/v0.24.0/. Somehow 0.24 is already the stable version.. Their API is not BC compatible, I spent whole debugging why some APIs are not found in the scope..